Automattic/wp-calypso
# Help-Center: enable help-center for all proxied users to make testing easier

#### Changes proposed in this Pull Request

* This enables the help-center for every a8cians to make testing easier. 
* It indicates that it's only available for a8cians to prevent panic :D 

#### Testing instructions
1. Behind a proxy: visit the editor and you should see the help center.
2. Not: not.
